.github/workflows/pull-request.yml

```
@@ -224,12 +243,14 @@ jobs:
           token: ${{ secrets.CODECOV_TOKEN }}
           verbose: true
           fail_ci_if_error: false
+          files: './coverage/lcov.info'
           name: '${{env.CODECOV_UNIQUE_NAME}}'
+          fail_ci_if_error: false
```

yamllint (1.35.1)

[error] duplication of key "fail_ci_if_error" in mapping

(key-duplicates)